packages/volto/src/components/theme/Icon/Icon.jsx

```diff
@@ -56,8 +56,9 @@ const Icon = ({
     onClick={onClick}
     id={id}
     aria-hidden={ariaHidden}
+    aria-label={title || ''}
```

Comment:
Do we really want aria-label as empty string in every current usage of this component, since we just fix the Sharing view here? What if we changed it to 
```suggestion
    aria-label={title}
```
so that aria-label is not rendered if title is not given? @plone/volto-accessibility